app/frontend/components/common/nufiScreenLink.tsx

```diff
@@ -0,0 +1,11 @@
+import {h} from 'preact'
+
+const NufiScreenLink = () => {
+  return (
+    <a href="http://adalite.io/nufi" target="_blank" rel="noopener">
+      Nufi
```

Comment:
shouldn't this be spelled `NuFi`?
```suggestion
      NuFi
```